targets/cli/cmake/build_web.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 set -e
 cd ../../web
+npm install
```

Comment:
if the esp32 target hasn't been built yet, `npm install` will not have been run. it's present in that copy of this script, but not this copy.